utils/ort/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -68,20 +68,22 @@ private val YEAR_RANGE_REGEX = "(?=.*)\\b([\\d]{4})( *- *)([\\d]{4}|[\\d]{2}|[\\
 private val PARTS_COMPARATOR = compareBy<Parts>({ it.owner }, { prettyPrintYears(it.years) }, { it.prefix })
 
 private fun getYearRanges(years: Collection<Int>): List<Pair<Int, Int>> {
-    fun divideAndConquer(years: IntArray, start: Int = 0, end: Int = years.size - 1): List<Pair<Int, Int>> {
-        if (end < start) return emptyList()
+    val yearRanges = mutableListOf<Pair<Int, Int>>()
```

Comment:
I actually had it like that before, but then decided that it would be nice to have the `yearRanges` variable as the "embracing" term in the function. Anyway, as you mentioned it, I'll change it back.